app/templates/_footer_categories.html

```diff
@@ -33,34 +33,34 @@
         <a href="https://www.gov.uk/government/organisations/crown-commercial-service">Crown Commercial Service</a>
       </li>
     </ul>
-    </div>
-    <div class="footer-suppliers">
-        <h2>
-            Guidance
-        </h2>
-        <ul>
-            <li>
-                <a href="https://www.gov.uk/guidance/digital-marketplace-buyers-guide">Digital Marketplace buyers’ guide</a>
-            </li>
-            <li>
-                <a href="https://www.gov.uk/guidance/digital-marketplace-suppliers-guide">Digital Marketplace suppliers’ guide</a>
-            </li>
-            <li>
-                <a href="https://www.gov.uk/guidance/g-cloud-buyers-guide">G-Cloud buyers’ guide</a>
-            </li>
-            <li>
-                <a href="https://www.gov.uk/guidance/g-cloud-suppliers-guide">G-Cloud suppliers’ guide</a>
-            </li>
-            <li>
-                <a href="https://www.gov.uk/guidance/digital-outcomes-and-specialists-buyers-guide">Digital Outcomes and Specialists buyers’ guide</a>
-            </li>
-            <li>
-                <a href="https://www.gov.uk/guidance/how-to-hire-user-research-labs-on-the-digital-marketplace">Digital Outcomes and Specialists user research labs buyers' guide</a>
-            </li>
-            <li>
-                <a href="https://www.gov.uk/guidance/digital-outcomes-and-specialists-suppliers-guide">Digital Outcomes and Specialists suppliers’ guide</a>
-            </li>
-        </ul>
-    </div>
-    <hr/>
+  </div>
+  <div class="footer-suppliers">
+    <h2>
+        Guidance
```

Comment:
You're right. But I'm handing in my badge as the indentation police. It's not a fun job. Merging.